test/src/hitbox_test.cpp

```diff
@@ -23,3 +24,24 @@ TEST (HitboxTest, simple_hitbox_overlapping_false) {
     bool actual = hitbox1.is_overlapping(hitbox2);
     EXPECT_EQ(expected, actual);
 }
+
+TEST (HitboxTest, constructor_1) {
+    Hitbox hitbox1 = Hitbox(0, 0, 100, 100);

```

Comment:
If `x` and `y` got swapped you wouldn't know with this test case. It would be better to use all different numbers, `Hitbox(1, 2, 3, 4);` would suffice.